Backend/Backend.Tests/NotificationTests.cs

```diff
@@ -10,6 +10,8 @@
 using Microsoft.Extensions.Logging;
 using Microsoft.VisualBasic;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.AspNetCore.Http;
+using static Backend.Infrastructure.FileStorageHelper;
 using MockQueryable.Moq;
```

Comment:
Prettyu sure this entire file is just the older version, it would be better to just keep the current version and not add any changes to this one

Reply:
I have changed the notification services that is why I have to update the tests